apps/labrinth/src/database/models/user_item.rs

```diff
@@ -668,6 +668,16 @@ impl DBUser {
             .execute(&mut **transaction)
             .await?;
 
+            sqlx::query!(
+                "
+                DELETE FROM uploaded_images
+                WHERE owner_id = $1
+                ",
+                id as DBUserId,
+            )
+            .execute(&mut **transaction)
+            .await?;
```

Comment:
This `DELETE` also removes images from mod versions associated with the deleted user. By this point, those versions have been transferred to the deleted user account, not removed entirely like this. Is this the behavior we want? To me, it seems more sensible to keep the versions as intact as possible to make data recovery easier when needed.

Reply:
Done, and did it for a couple other things too